backend/tracim_backend/config.py

```diff
@@ -70,13 +68,23 @@ def __init__(self, settings: typing.Dict[str, typing.Any]):
         # with object in some context
         self.settings = settings.copy()
         self.config_naming = []  # type: typing.List[ConfigParam]
-        logger.debug(self, "CONFIG_PROCESS:1: load config from settings")
+        logger.debug(self, "CONFIG_PROCESS:1: load enabled apps")
```

Comment:
We do need to load app_enabled config first, then load app and them we can do load/check like before.